src/index.js

```diff
@@ -1,6 +1,8 @@
 import DropZone from './classes/drop_zone';
 import Form from './classes/form';
 import './main.css';
+import {serviceWorker} from './sw';
```

Comment:
super nit, but can we move this one line above before the css import?

Reply:
would you like me to add a prettier ext. to keep the formatting consistent?

Reply:
oh shoot, yeah that would be great. already have the `.prettierrc` file

Reply:
I've ordered this as requested, but to lint the file order I would recommend taking a look at adding [prettier-plugin-sort-imports](https://www.npmjs.com/package/@trivago/prettier-plugin-sort-imports) or ESLint. The latter we use at work, and there are some quite good plugins.